services/deeplink/src/pages/buckets/share.tsx

```diff
@@ -19,7 +19,18 @@ const Margin = styled.div`
   height: 25px;
 `;
 
-const ShareBucket = ({ search }) => {
+interface ShareBucketProps {
+  search: {
+    bucket?: string;
+    multiaddr?: string;
+    invitationid?: string;
+    key?: string;
+  };
+}
+
+const ShareBucket: React.FC<ShareBucketProps> = ({
+  search,
+}: ShareBucketProps) => {
   const { bucket, multiaddr, invitationid, key } = search;
   const deeplink = `space://buckets/share?bucket=${bucket}&multiaddr=${multiaddr}&invitationid=${invitationid}&key=${key}`;
```

Comment:
Was this meant to be the "Open in Space" handler? I'm thinking now that the architecture has changed if we should create a new link for this and just send the file hash and key so that the daemon can download the file, and store it in their "Shared with Me" thread (if we do it that way we might have to adjust the shared with me schema to support looking up file by hash instead of bucket + path).